### ISSUE TITLE
[TC6, Foci] Craft focus from bottom up

### ISSUE THREAD

**Issue by AnrDaemon** [Azanor/thaumcraft-suggestions#222]
The new foci crafting system is entertaining, but the top to bottom direction really irk me.

Feels much like I'm aiming the focus at myself.

**Comment by Alcamore** [comment]
I agree, starting from the bottom would feel more like you're constructing the spell from the ground up.
